src/renderer.tsx

```diff
@@ -34,6 +34,7 @@ import React from 'react'
 import { render } from 'react-dom'
 import { hot } from 'react-hot-loader' // has to stay first
 import { Provider } from 'react-redux'
+import log from 'electron-log'
```

Comment:
We at javascript are get used to console.log, without imports.

I think it would be better to assign log to console:
console.log = log.log;